README.md

```diff
@@ -30,6 +30,39 @@ To build an individual package in response to changes within the `src` directory
 ```sh
 yarn dev
 ```
+
+### Testing
+Tests are run with [Mocha](https://mochajs.org/), [Chai](https://www.chaijs.com/), and [Sinon](https://sinonjs.org/) with the [Karma test runner](https://karma-runner.github.io/latest/index.html) in a browser environment. The following browsers are currently tested:
+
+- Chrome
+
+To run tests for a specific web component package, navigate to its directory and use one of the following commands based on the desired testing flow:
+
+- To run the tests for a specific package once:
+```sh
+yarn test
+```
+- To run the tests for a specific package and watch the `src` and `test` directories to rerun the tests after any changes:
+```sh
+yarn tdd
```

Comment:
I'm assuming `yarn tdd` is not intended to be used because people will use it when they are developing a specific component. If that's the case, maybe make it clear that `yarn test` works when run at the top level, but `yarn tdd` does not.

Reply:
Clarified this in the README